buildscripts/common.props

```diff
@@ -32,11 +32,11 @@
 	</PropertyGroup>
 
 	<PropertyGroup Condition="'$(TargetFramework)|$(Configuration)'=='net461|Debug'">
-		<DefineConstants>$(DiagnosticsConstants);$(DesktopClrConstants)</DefineConstants>
+		<DefineConstants>$(DiagnosticsConstants);$(DesktopClrConstants);FEATURE_TEST_SERILOGINTEGRATION</DefineConstants>
 	</PropertyGroup>
 
 	<PropertyGroup Condition="'$(TargetFramework)|$(Configuration)'=='net461|Release'">
-		<DefineConstants>$(DesktopClrConstants)</DefineConstants>
+		<DefineConstants>$(DesktopClrConstants);FEATURE_TEST_SERILOGINTEGRATION</DefineConstants>
```

Comment:
Shouldn't this also be defined for netstandard? It was previously used to disable these tests on .NET 3.5 and 4.0.